rotors_gazebo_plugins/src/gazebo_bag_plugin.cpp

```diff
@@ -384,12 +388,21 @@ void GazeboBagPlugin::LogWrenches(const common::Time now) {
     wrench_msg.header.frame_id = collision1_name + "--" + collision2_name;
     wrench_msg.header.stamp.sec = now.sec;
     wrench_msg.header.stamp.nsec = now.nsec;
+#if GAZEBO_MAJOR_VERSION >= 7
```

Comment:
when did this change? I think jenkins uses gazebo7 for 16.04 and it does not compile.